Jellyfin.Plugin.KodiSyncQueue/Configuration/PluginConfiguration.cs

```diff
@@ -20,19 +20,14 @@ public PluginConfiguration()
 
         public bool IsEnabled { get; set; }
 
-        [XmlElement("tkMovies")]
         public bool TkMovies { get; set; }
 
-        [XmlElement("tvTVShows")]
         public bool TkTvShows { get; set; }
 
-        [XmlElement("tkMusic")]
         public bool TkMusic { get; set; }
 
-        [XmlElement("tkMusicVideos")]
         public bool TkMusicVideos { get; set; }
 
-        [XmlElement("tkBoxSets")]
         public bool TkBoxSets { get; set; }
```

Comment:
I am not sure if that has any effect on already existing saved configs of the plugin. So I am not 100% if this is something we want to do. I would like another opinion on this though.
The rest LGTM

Reply:
@h1dden-da3m0n it does have effect on the user conflagration. but the user need to updated the config again. without this change https://github.com/jellyfin/jellyfin-plugin-kodisyncqueue/pull/46/commits/4cd285b32bc5dd821389dece08fd03288b3ddf9a the user is unable to read the saved config

Reply:
on this comment https://github.com/jellyfin/jellyfin-plugin-kodisyncqueue/pull/46/commits/7d7440e532175952ba942cc2e161a6b55e60a2b9 the user get like this 

**kodisyncqueue setting page**
![Screenshot 2021-05-20 19543912](https://user-images.githubusercontent.com/32230989/119018905-4a2ae680-b9a5-11eb-9296-5dca4c458bd5.jpg)
**saved config Jellyfin.Plugin.KodiSyncQueue.xml**
![Screenshot 2021-05-20 19480312](https://user-images.githubusercontent.com/32230989/119018913-4d25d700-b9a5-11eb-9528-b24bcbe304c3.jpg)

Reply:
Okay, I checked out your branch and just checked one more time to be sure.
Removing the `XmlElement` annotations will render any existing configuration as unreadable since our XML serializer is case sensitive.
To prevent that you could just restore the `XmlElement` in the `PluginConfiguration.cs` and keep the rest of your changes from 4cd285b, which will keep the config page operational (i.e. you can still save and load the config).

However, personally I have no strong opinion on keeping or removing the annotations, I think we have a mix within the plugin configs already, however most seem to not bother with annotations. (i.e. most plugins just use the property names as the config keys so I tend to agree with the removal of the annotations)

I will sleep over this one more time and likely merge this tomorrow, unless some other team member disagrees or does it b4 me.

Thank you again for your contribution!

Reply:
Thanks